lib/ssf/client.rb

```diff
@@ -27,69 +27,21 @@ def send_to_socket(message)
       @socket.send(message, 0)
     end
 
-    def new_root_span(operation: '', tags: {})
+    def new_span(operation: '', tags: {}, trace_id: nil, parent_id: nil)
```

Comment:
Could we call this `start_span`? The signature doesn't actually match what we currently use in Go (because that's tied to OpenTracing), but it would be nice to keep them as similar as possible.